test/extensions/filters/common/ext_authz/check_request_utils_test.cc

```diff
@@ -106,15 +106,17 @@ class CheckRequestUtilsTest : public testing::Test {
   NiceMock<Envoy::StreamInfo::MockStreamInfo> req_info_;
   Buffer::InstancePtr buffer_;
   const std::string cert_data_{"cert-data"};
+  const absl::string_view requestedServerName = "server.name";
```

Comment:
Nit: based on https://github.com/envoyproxy/envoy/blob/main/STYLE.md

```suggestion
  const absl::string_view requested_server_name_{"server.name"};
```